documentation/tls-best-practices.md

```diff
@@ -0,0 +1,71 @@
+---
+title: IRCv3 TLS Best Practices
+layout: spec
+copyrights:
+  -
+    name: "William Pitcock"
+    period: "2016"
+    email: "nenolod@dereferenced.org"
+---
+
+This document specifies a series of baseline security practices for managing TLS on IRC networks.
+It should be considered non-normative.
+
+
+# Clients
+
+* Clients SHOULD upgrade the default port to 6697 and enable TLS per IETF [RFC 7194][rfc7194].
+
+  [rfc7194]: https://tools.ietf.org/html/rfc7194
+
+* Clients SHOULD allow a user to downgrade to plaintext if they explicitly configure the client
+  to do so.  A suggested `/server` command may be something like:
+
+    /server -insecure irc.example.com 6667
```

Comment:
I'm not so sure we should be recommending client interface implementation details.